src/Components/Track.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../Styling/Track.css';
+
+
+function Track() {
+  return (
+    <div className="Track">
+        <div className="Track-information">
+            <h3> 
+                {/* {<!-- track name will go here -->}  */}
```

Comment:
I’ll accept it just this one time, but for the future, you should NEVER merge unfinished code. It’s best to code the functionality/UI piece by piece to avoid confusion, limit unnecessary code and complexity, and potential for regression. Look up the term YAGNI(You Aint Gonna Need It). This term is basically just a way to say don’t overplan by implementing features you’ll use in the future. chances are you aint gonna need it and/or it’ll add unneeded complexity to you app in the present.